tools/csa/types.go

```diff
@@ -0,0 +1,5 @@
+package csa
+
+type Settings struct {
```

Comment:
Same as before
I don't think that is a good idea because adding those parameters on the config when we already have it on the commands flags is to duplicate everything.
I meant, imagine that you want to add every types as command in
```go
talos run csa
```
Like all this commands
```bash
  ImageName: '#Required for api scan type; Running Docker image'
  Network: '#Optional; Change if network is other than bridge'
  ZAPConfigFileName: '#Required; ZAP config file location'
  OpenAPIConfigFileName: '#Required; Name of OpenAPI Config file'
  ScanType: '#Required; one of website or api'
  auth:
    website:
      accesstoken: '#Required for website; AccessToken stands for authentication header,
        sent in every request'
    api:
      ClientID: '#Required for api; Will be used to get authorization from API'
      ClientSecret: '#Required for api; Will be used to get authorization from API'
      GrantType: '#Required for api; Will be used to get authorization from API'
      Username: '#Required for api; Will be used to get authorization from API'
      Password: '#Required for api; Will be used to get authorization from API'
```

Thats not make sense to have multiple options one for config and another for command line. We will need to do a priority config from the file instead of the command line?